examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Is this output deletion intentional? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yes, since we do not want mentions of "IonQ Device" in the NBI.

Reply:
In that case, should it have been changed to `[BraketBackend[Aspen-M-3], BraketBackend[Aria], BraketBackend[Lucy]]`?

Reply:
I think there is some question as to having the execution output included. For now, we've opted to remove it, but we can always decide what we want to do at our next discussion.